pickfirst.go

```diff
@@ -64,25 +68,36 @@ func (b *pickfirstBalancer) HandleSubConnStateChange(sc balancer.SubConn, s conn
 }
 
 func (b *pickfirstBalancer) ResolverError(err error) {
+	if b.state == connectivity.TransientFailure {
+		// Set a failing picker if we don't have a good picker.
+		b.cc.UpdateBalancerState(connectivity.TransientFailure,
+			&picker{err: status.Errorf(codes.Unavailable, "name resolver error: %v", err)},
```

Comment:
This will fail wait-for-ready RPCs, because we check the value of the returned error.